QUEST.tsv

```diff
@@ -1155,20 +1155,20 @@ QUEST_20150317_001154	With her hand to her chin, the old woman gave a puzzling r
+QUEST_20150317_001158	Anyhow, as Modestas and the Old Lady parted ways, he knew he now had a dangerous power guaranteeing his victory at any time in the near future. {nl} He walked, looking for the next person to bet huge on. As he wondered how much to bet on his next move, he heard the sound of running footsteps behind him.
+QUEST_20150317_001159	Modestas looked back, it was his good old friend Sigfried on horseback riding towards him. {nl} Sigfried and Modestas knew each other, as both had been together through the highs and lows of gambling. They greeted each other, and talked about where they were going. To walk with his good friend Modestas, Sigfried got off his horse. They walked at a leisurely pace, with idle chatter along the way. {nl}
+QUEST_20150317_001160	However, before long, Modestas could no longer continue walking with Sigfried. He was fully aware of his deadline, and had no time for trivial activities with Sigfried. {nl}
+QUEST_20150317_001161	Even during their normal everyday talk, Modestas and Sigfried unconsciously made big and small bets. Eventually, one would utter the word, "bet?" without knowing he did. {nl} Both Sigfried and Modestas weren't rich, therefore, there was no way this bet could be big. Knowing this could trigger the curse, Modestas really wanted to simply tell Sigfried he's gotta go. {nl} In the end, Modestas uttered these words. {nl}
+QUEST_20150317_001162	"Are you not busy?" You seemed busy on your horse before you passed by for a talk. You should stop worrying about me and go." {nl} Sigfried, hearing Modestas's words, replied, {nl} "I was busy, but I still have time to walk with you for another 10 minutes or so. This horse might look look average, but when it comes to it he can go pretty fast." {nl}
+QUEST_20150317_001163	Seeing Sigfried not budge, Modestas took another shot at persuading him to go, {nl} Wouldn't it be better to be early as opposed to be on time?" {nl} {nl} Hearing this, Sigfried became adamant, {nl} "Do you not believe what I just said? Or are you wilfully ignoring it all?" {nl} A disconcerted Modestas said, {nl} "No, I didn't mean it like that……" {nl}
+QUEST_20150317_001164	However, before Modestas could say any more, Sigfried said, "If you don't believe me that much, why not have a bet?" {nl} {nl} Naturally, the words, "Sure. what do you want to bet?" rose up in Modestas' throat.
+QUEST_20150317_001165	Considering Modestas' habit, to simply not immediately accept a gamble took superhuman willpower from him. {nl} "What…  what kind of bet?" {nl} "We were going to bet on whether my horse is fast or not." {nl} Modestas desperately fought in his inner gambler.
```

Comment:
"fought in his inner gambler" -> "fought his gambler nature"?
I'm not familiar with such a use of "in" in this context.